### PR TITLE
LANG-1495 Update EnumUtils.java

### PR DESCRIPTION
**Pull request by vlcheong**
Add Overloading Methods getEnum and getEnumIgnoreCase To EnumUtils

### CODE REVIEW

File: src/main/java/org/apache/commons/lang3/EnumUtils.java [hunk 1]
```diff
@@ -119,13 +119,29 @@ public EnumUtils() {
...
+     * <p>Gets the enum for the class, returning {@code defaultEnum} if not found.</p>
+     *
+     * <p>This method differs from {@link Enum#valueOf} in that it does not throw an exception
+     * for an invalid enum name.</p>
+     *
+     * @param <E> the type of the enumeration
+     * @param enumClass   the class of the enum to query, not null
+     * @param enumName    the enum name, null returns default enum
+     * @param defaultEnum the default enum
+     * @return the enum, default enum if not found
```
Comment (Stzx) [inline src/main/java/org/apache/commons/lang3/EnumUtils.java]: Provide a `@since` tag for the new method.

Comment (vlcheong) [reply src/main/java/org/apache/commons/lang3/EnumUtils.java]: @Stzx, may I know what version number should I put ?

Comment (Stzx) [reply src/main/java/org/apache/commons/lang3/EnumUtils.java]: `3.10` :)

Comment (vlcheong) [reply src/main/java/org/apache/commons/lang3/EnumUtils.java]: @Stzx, Thanks for your guidance.

**Pull request merged**